README.md

```diff
@@ -93,47 +93,31 @@ if __name__ == "__main__":
 ### Client
 
 ```ts
-import { Client } from "@modelcontextprotocol/sdk/client/index.js";
-import { StdioClientTransport } from "@modelcontextprotocol/sdk/client/stdio.js";
-import { ChatOpenAI } from "@langchain/openai";
-import { createReactAgent } from "@langchain/langgraph/prebuilt";
-import { loadMcpTools } from "@langchain/mcp-adapters"
+import { MultiServerMCPClient } from '@langchain/mcp-adapters';
+import { ChatOpenAI } from '@langchain/openai';
+import { createReactAgent } from '@langchain/langgraph/prebuilt';
 
 // Initialize the ChatOpenAI model
-const model = new ChatOpenAI({ modelName: "gpt-4" });
+const model = new ChatOpenAI({ modelName: 'gpt-4' });
 
-// Create transport for stdio connection
-const transport = new StdioClientTransport({
-  command: "python",
-  args: ["math_server.py"]
-});
+// Create client and connect to server using the adapter
+const client = new MultiServerMCPClient();
```

Comment:
please revert this to the current README - we want to show how people can use w/ MCP SDK

Reply:
Reverted it